src/lib/swapper/swappers/ThorchainSwapper/types.ts

```diff
@@ -32,6 +33,7 @@ export type ThornodeQuoteResponseSuccess = {
   outbound_delay_seconds: number
   router: string
   slippage_bps: number
+  streaming_slippage_bps: number
```

Comment:
Do we want to add `streaming_swap_seconds` as well, so we can add it as an informative metric as a follow-up? Since the biggest differentiator of streaming swaps from a user perspective is how long they will take, we probably want to have that info handy

Reply:
We do, but in the this PR. It's been discussed in a "close follow up" task